packages/fluentui/react-northstar/src/components/Avatar/AvatarLabel.tsx

```diff
@@ -0,0 +1,111 @@
+import { Accessibility } from '@fluentui/accessibility';
+import {
+  ComponentWithAs,
+  getElementType,
+  useUnhandledProps,
+  useAccessibility,
+  useFluentContext,
+  useStyles,
+  useTelemetry,
+} from '@fluentui/react-bindings';
+import * as customPropTypes from '@fluentui/react-proptypes';
+import * as PropTypes from 'prop-types';
+import * as React from 'react';
+
+import {
+  childrenExist,
+  createShorthandFactory,
+  UIComponentProps,
+  ChildrenComponentProps,
+  ContentComponentProps,
+  commonPropTypes,
+  ColorComponentProps,
+  rtlTextContainer,
+  SizeValue,
+} from '../../utils';
+
+import { BoxProps } from '../Box/Box';
+
+import { ShorthandValue, FluentComponentStaticProps } from '../../types';
+import { labelClassName } from '../Label/Label';
+
+export interface AvatarLabelProps
```

Comment:
so apart from the `Image` slot that we decided to remove, `icon` , `iconPosition`, `fluid`, `circular` props will also result in breaking changes. Are we happy with that ?

Reply:
Yeah, looks like it because it wasn't working really with Avatar anyway. Pretty much following @layershifter suggestion and making it as a copy of `Box` component

Reply:
Might want to make a note in PR description then. You should never underestimate how people will use 'things that weren't working anyway` after reading them on a docsite :-)